nativeruntime/src/main/java/org/robolectric/nativeruntime/DefaultNativeRuntimeLoader.java

```diff
@@ -95,10 +95,9 @@ private void maybeCopyIcuData(TempDirectory tempDirectory) throws IOException {
     } catch (IllegalArgumentException e) {
       return;
     }
-    Path icuPath = tempDirectory.create("icu");
     Path icuDatPath = tempDirectory.getBasePath().resolve("icu/icudt68l.dat");
     Resources.asByteSource(icuDatUrl).copyTo(Files.asByteSink(icuDatPath.toFile()));
-    System.setProperty("icu.data.path", icuPath.toAbsolutePath().toString());
+    System.setProperty("icu.data.path", icuDatPath.toAbsolutePath().toString());
```

Comment:
Would it make sense to update `DefaultNativeRuntimeLoaderTest#extracts_fontsAndIcuData` to check the value of `System.getProperty("icu.data.path")`?

Reply:
This is a good suggestion, will look into this tomorrow.